### PR TITLE
Added the first web page

### LINKED ISSUE #15: Create homepage for project
Create HTML and CSS for project demo 1, including:

- Project name + logo
- Description of the project
- Team members names

### PR DESCRIPTION
**Pull request by shirazyomtov**
1.Create a new app directory called 'main' and register it by modifying the settings.py file.
 2.Create a new function to render our home page in 'views.py' and create a new route in the 'urls.py' file. 
3.Added a static home page template.
fixes #15

### CODE REVIEW

**Review by Davidco94 — APPROVED** [review]
LGTM

**Review by shacharlevy — APPROVED** [review]
LGTM

File: static/images/HighTeach_logo.jpeg
Comment (natifridman) [inline static/images/HighTeach_logo.jpeg:1]: We already have the logo, so maybe we should move it to the static folder?

**Review by natifridman — APPROVED** [review]
Looks good. Few comments though for the next PR :)
- Try not to use `:` in the short commit message for no reason. That's what people will see in the repo.
- Please try not to force push during the review, it makes the review harder. You can squash commits at the end of the review.  
- This PR is big and could be split into 2 smaller PRs that could help the reviewer (consider this for next time)
  - The static folder with all the images
  - The Django app with all the generated files

**Pull request merged**